packages/create-remix/index.ts

```diff
@@ -285,41 +274,27 @@ async function templateStep(ctx: Context) {
     start: "Template copying...",
     end: "Template copied",
     while: async () => {
-      let destPath = path.resolve(process.cwd(), ctx.cwd);
-      await ensureDirectory(destPath);
-      await copyTemplate(template, destPath, {
+      await ensureDirectory(ctx.tempDir);
+      if (ctx.debug) {
+        info(`Extracting template to temp directory: ${ctx.tempDir}`);
+      }
+
+      // TODO: Optimization - if template is just a local directory (not a
+      // local tarball etc.), just use that as ctx.tempDir for the rest of
+      // the pipeline and avoid copying it to a temp directory
+
+      await copyTemplate(template, ctx.tempDir, {
         debug: ctx.debug,
         token: ctx.token,
         async onError(err) {
-          let cwd = process.cwd();
```

Comment:
No need to do this cleanup anymore - it's extracting into a temp dir which the OS should cleanup automatically.